cores/arduino/WInterrupts.c

```diff
@@ -62,48 +62,23 @@ void attachInterrupt(uint32_t pin, voidFuncPtr callback, uint32_t mode)
 #else
   EExt_Interrupts in = digitalPinToInterrupt(pin);
 #endif
-  if (in == NOT_AN_INTERRUPT) return;
+  if (in == NOT_AN_INTERRUPT || in == EXTERNAL_INT_NMI)
```

Comment:
Why did we make this change? Or did we possibly miss a bug fix that is in my repo?